experiments/mountain_car_plots.py

```diff
@@ -0,0 +1,269 @@
+"""Heatmaps and reward plotting code for debugging MountainCar."""
+
+from functools import partial
```

Comment:
"Use import statements for packages and modules only, not for individual classes or functions." https://google.github.io/styleguide/pyguide.html#22-imports